### PR TITLE
Merging install.sh and init.sh

### PR DESCRIPTION
**Pull request by pasthorizon**
Merged **install.sh** and **init.sh** to eliminate the need to run two different scripts. The need for manually stopping the `docker-compose up` is eliminated by redirecting the logs to text file and periodically running a pattern match for "_mysqld: ready for connections_".
The command `docker-compose up` is given from a different terminal. When the pattern matches, we stop the process and the newly created terminal is closed. We have appended the **init.sh** script to **install.sh** , which ensures its running as soon as the process is killed.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -120,6 +123,63 @@ echo "When output says DB is ready to receive connections, hit CTRL+C and move o
 
 sleep 5 #So that user reads the above lines properly
 
-docker-compose up
+gnome-terminal -- docker-compose up
```
Comment (grapheo12) [inline install.sh:126]: Do not expect that people will be using gnome-terminal. I use Konsole myself.

Comment (grapheo12) [reply install.sh:126]: Can you try with `docker-compose up -d`? Then I guess you'll not have to rely on a separate window in the terminal.

**Review by grapheo12 — CHANGES_REQUESTED** [review]
Good work! But there are some necessary changes. Please look into them tonight. I'll merge tomorrow.

**Comment by grapheo12** [comment]
Is `init.sh` unnecessary now? Please delete it if that is the case.

**Comment by pasthorizon** [comment]
@grapheo12 Changes made:)

**Comment by grapheo12** [comment]
@pasthorizon There is a merge conflict due to #39. Please resolve it. Looks good to me otherwise.
After removing conflict, please check if:
1. Pause resume functionality works as expected.
2. Your feature also works as expected.
Then commit. I'll merge.

**Comment by pasthorizon** [comment]
Hey @grapheo12 I have made the changes. Kindly look into it

**Comment by grapheo12** [comment]
Hey! There are still merge conflicts remaining. There is a whole chunk of commented out code. Please see.
Don't worry about midevals. I'll handle them.

**Pull request merged**